src/views/app_panel.dot

```diff
@@ -2,6 +2,7 @@
   <div class="favorite_poi_panel__container">
     {{= this.favoritePanel.render() }}
     {{= this.poiPanel.render() }}
+    {{= this.directionPanel.render() }}
```

Comment:
to render only if directions are enabled

Reply:
sure